code/iomethods.py

```diff
@@ -518,5 +544,132 @@ def decode_time_from_string(timeString):
+                 lat: an integer representing the time interval found in the file's metadata
+                 t: A 3 dimensional numPy array holding temperatures in Kelvin.
+                    The first dimension is time and the second and third dimensions are longitude/latitude
+                 fileName: The name of the file to be written to
+                 dirName: Path to the directory to be written to
+        Returns:: None. It writes to a netCDF file with extension .nc
+    '''
+
+    ncdf = netCDF4.Dataset(dirName + '/' + fileName + '.nc', "w", format="NETCDF4")
+    ncdf.Conventions = "COARDS"  # Set global attributes
+    ncdf.calendar = "standard"
+    ncdf.comments = "File"
+    ncdf.model = "geos/das"
+    ncdf.center = "gsfc"
+
+    time = ncdf.createDimension("time", None)  # Create and set dimensions
+    longitude = ncdf.createDimension("longitude", size=9896)
+    latitude = ncdf.createDimension("latitude", size=3298)
+
+    longitudeVariable = ncdf.createVariable("longitude", "double", dimensions=("longitude",))  # Create and set variables
+    longitudeVariable.units = "degrees_east"                                                   # and their local attributes
+    longitudeVariable.long_name = "Longitude"
+
+    latitudeVariable = ncdf.createVariable("latitude", "double", dimensions=("latitude",))
+    latitudeVariable.units = "degrees_north"
+    latitudeVariable.long_name = "Latitude"
+
+    timeVariable = ncdf.createVariable("time", "double", dimensions=("time",))
+
+    ch4Variable = ncdf.createVariable("ch4", np.float, dimensions=("time","latitude","longitude",))
+    ch4Variable.comments = "Unknown1 variable comment"
+    ch4Variable.long_name = "IR BT (add 75 to this value)"
+    ch4Variable.units = ""
+    ch4Variable.grid_name = "grid01"
+    ch4Variable.grid_type = "linear"
+    ch4Variable.level_description = "Earth surface"
+    ch4Variable.time_statistic = "instantaneous"
+    ch4Variable.missing_value = float(330)
+
+    mergFileName = fileName  # Parse file name to get date and time to set as an attribute of timeVariable
+    year = mergFileName[5:9]    # Fix this later to use the functions in iomethods
+    month = mergFileName[9:11]
+    day = mergFileName[11:13]
+    hour = mergFileName[13:15]
+
+    timeVariable.units = "hours since " + year + "-" + month + "-" + day + " " + hour
+
+    longitudeVariable[:] = lon  # Assign numPy arrays to netCDF variables
+    latitudeVariable[:] = lat
+    ch4Variable[:,:,:] = t
+
+    ncdf.close()
+
+
+def read_MERG_pixel_file(path, shape=(2, 3298, 9896), offset=75.):
+    '''
+    Read MERG brightness temperature from binary file. Thanks to Brian Wilson for this contribution.
+    File contains two large arrays (2 time epochs: on the hour and the half hour)
+    of temperature (Kelvin) as an unsigned integer byte, offset by 75 so it will fit in the 0-255 range.
+    For documentation, see http://www.cpc.ncep.noaa.gov/products/global_precip/html/README.
```

Comment:
Include in doc string the assumption that the binary file was unmodified when downloaded. i.e. the dims that are hardcoded in will be this always.